src/thor/costmatrix.cc

```diff
@@ -152,70 +150,87 @@ void CostMatrix::SourceToTarget(Api& request,
   SetSources(graphreader, source_location_list, time_infos);
   SetTargets(graphreader, target_location_list);
 
+  // Update hierarchy limits
+  if (!ignore_hierarchy_limits_)
+    ModifyHierarchyLimits();
+
   // Perform backward search from all target locations. Perform forward
   // search from all source locations. Connections between the 2 search
   // spaces is checked during the forward search.
-  int n = 0;
+  uint32_t n = 0;
   while (true) {
-    // Iterate all target locations in a backwards search
```

Comment:
why remove things like this comment? i know its clear to you and me but its nice to have headings on code for newcomers

Reply:
ok..